skoolkit/snapinfo.py

```diff
@@ -404,6 +405,8 @@ def main(args):
     group = parser.add_argument_group('Options')
     group.add_argument('-b', '--basic', action='store_true',
                        help='List the BASIC program')
+    group.add_argument('-v', '--vars', action='store_true',
+                       help='List variables')
```

Comment:
Lines moved below -t/--find-text, assuming you meant alphabetical order with respect to the short argument flags.